pkg/runctl/controller.go

```diff
@@ -354,6 +373,9 @@ func (c *Controller) syncHandler(key string) error {
 		if err == nil {
 			err = c.changeState(pipelineRun, api.StateFinished)
 		}
+		if err == nil {
+			finishedRuns.Store(key, true)
```

Comment:
I am just wondering if this bookkeeping is really warranted for something non-functional like: more selective logs.

Reply:
What do you mean by warranted? For logging it is OK with respect to reliability. If we would use `finishedRuns` for functional processing where reliable bookkeeping  is urgent we should review this again and maybe think about a different approach.

Reply:
I am asking if it is justified. Why do this effort for printing log lines that no one actually needs?

Reply:
Agree, once we can configure the verbosity in a live cluster on the fly we don't need the entries all the time